_benchmark/commands/execute-test.md

```diff
@@ -0,0 +1,178 @@
+|           Heap used for points |                      |    0.5793 |     MB |
+|    Heap used for stored fields |                      |  0.643608 |     MB |
+|                  Segment count |                      |        97 |        |
+|                 Min Throughput |         index-append |   31925.2 | docs/s |
+|              Median Throughput |         index-append |   39137.5 | docs/s |
+|                 Max Throughput |         index-append |   39633.6 | docs/s |
+|      50.0th percentile latency |         index-append |   872.513 |     ms |
+|      90.0th percentile latency |         index-append |   1457.13 |     ms |
+|      99.0th percentile latency |         index-append |   1874.89 |     ms |
+|       100th percentile latency |         index-append |   2711.71 |     ms |
+| 50.0th percentile service time |         index-append |   872.513 |     ms |
+| 90.0th percentile service time |         index-append |   1457.13 |     ms |
+| 99.0th percentile service time |         index-append |   1874.89 |     ms |
+|  100th percentile service time |         index-append |   2711.71 |     ms |
+|                           ...  |                  ... |       ... |    ... |
+|                           ...  |                  ... |       ... |    ... |
+|                 Min Throughput |     painless_dynamic |   2.53292 |  ops/s |
+|              Median Throughput |     painless_dynamic |   2.53813 |  ops/s |
+|                 Max Throughput |     painless_dynamic |   2.54401 |  ops/s |
+|      50.0th percentile latency |     painless_dynamic |    172208 |     ms |
+|      90.0th percentile latency |     painless_dynamic |    310401 |     ms |
+|      99.0th percentile latency |     painless_dynamic |    341341 |     ms |
+|      99.9th percentile latency |     painless_dynamic |    344404 |     ms |
+|       100th percentile latency |     painless_dynamic |    344754 |     ms |
+| 50.0th percentile service time |     painless_dynamic |    393.02 |     ms |
+| 90.0th percentile service time |     painless_dynamic |   407.579 |     ms |
+| 99.0th percentile service time |     painless_dynamic |   430.806 |     ms |
+| 99.9th percentile service time |     painless_dynamic |   457.352 |     ms |
+|  100th percentile service time |     painless_dynamic |   459.474 |     ms |
+
+----------------------------------
+[INFO] SUCCESS (took 2634 seconds)
+----------------------------------
+```
+
+## Options
+
+Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use-case.
+
+## General settings
+
+The following options shape how each test runs and how results are output.
+
+- `--test-mode`: Runs the given workload in test mode, which is useful when checking a workload for errors.
+- `--user-tag`: Defines user-specific key-value pairs to be used in metric record as meta information, for example, `intention:baseline-ticket-12345`.
```

Comment:
yes, it looks like "key-value pair" is the preferred formatting across the web. (with hyphen)